### PR TITLE
Add TheMaritimePack.netkan

### PR DESCRIPTION
**Pull request by plague006**
Breakup of #1786

### CODE REVIEW

File: NetKAN/TheMaritimePack.netkan [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "spec_version": 1,
+    "license": "BSD",
+    "$kref": "#/ckan/kerbalstuff/942",
+    "identifier": "TheMaritimePack",
```
Comment (dbent) [inline NetKAN/TheMaritimePack.netkan]: Nitpick, can you drop the definite article from the identifier?

**Comment by plague006** [comment]
Hadn't noticed that there's not a single identifier starting with "The"

File: NetKAN/MaritimePack.netkan [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "spec_version": 1,
+    "license": "BSD",
```
Comment (dbent) [inline NetKAN/MaritimePack.netkan]: From glancing as the `License.txt` in the distribution it looks like the `BSD-3-clause`.

**Comment by plague006** [comment]
Done

**Comment by dbent** [comment]
Tested manually, merging. Thanks.

Also thanks for putting the appropriate `Closes` comment so I don't have to track down the KerbalStuffBot PR.

**Pull request merged**